frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -165,12 +161,10 @@ const deleteDelegatedAdmin = (
                     <template #body="{ data }">
                         <button
                             class="btn btn-icon"
+                            title="Delete"
```

Comment:
Images should have `title` for the screen readers to read them, or else it would says 'image', which leads to confusion.

Reply:
I'm not sure that just "delete" is enough for the user to understand what the button will do

Reply:
It makes more sense. I have updated all other delete buttons as well. Thanks.